app/controllers/home_controller.rb

```diff
@@ -144,6 +144,7 @@ def account
     end
 
     @user = LinkedData::Client::Models::User.get(session[:user].id, include: 'all')
+    @all_ontologies = LinkedData::Client::Models::Ontology.all(ignore_custom_ontologies: true).map {|x| ["#{x.name} (#{x.acronym})", x.acronym]}
```

Comment:
I think here you can remove this `account` action here as it is the same as `show` in the `users_controller` 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/f7a383f2-e340-4cd6-a1a9-436b283817c8)